### ISSUE TITLE
Extend to another type of cnc controller

### ISSUE THREAD

**Issue by smtpl-iiotian** [TrakHound/MTConnect.NET-Haas#1]
Thanks @PatrickRitchie for this guide to make Haas Machine data available on MTConnect Agent using MTConnect.NET library.  
In the regards of same, can I refer it as a guide to route Mitsubishi cnc data to MTConnect Agent.  I will make use of Mitsubishi communication dll libraries. Will I be able to extend this method to x number of Mitsubishi cnc machines ?

**Comment by PatrickRitchie** [comment]
Yes you are welcome to use it as a guide for a Mitsubishi CNC and yes you can extend it as needed. Please let me know if you have any other questions.
